recipes/mvapich2/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set version = "3.0" %}
+
+package:
+  name: mvapich
+  version: {{ version }}
+
+source:
+  url: https://mvapich.cse.ohio-state.edu/download/mvapich/mv2/mvapich-{{ version }}.tar.gz
+  sha256: ee076c4e672d18d6bf8dd2250e4a91fa96aac1db2c788e4572b5513d86936efb
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+  ignore_run_exports:
+    - libhwloc
+    - libnl
+    - libnuma
+    - libzlib
+    - rdma-core
+
+requirements:
+  build:
+    - bison
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - {{ stdlib("c") }}
+    - make
+  host:
+    - libhwloc
+    - libnl  # [linux]
+    - libnuma  # [linux]
+    - libzlib
+    - rdma-core  # [linux]
+  run:
+    - mpi 1.0 mvapich
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - {{ stdlib("c") }}
+    - libhwloc
+    - libnl  # [linux]
+    - libnuma  # [linux]
+    - libzlib
+    - rdma-core  # [linux]
```

Comment:
`run` should only include dependencies that are _not_ inherited via `run_exports` from build/host dependencies.

```suggestion
```

Reply:
OK